README.md

```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/haiweiosu/cs5356.png?label=ready&title=Ready)](https://waffle.io/haiweiosu/cs5356)
```

Comment:
Sorry I made an error on that. Could you just cancel that pull request
please?

Thanks,
Haiwei

On Monday, September 21, 2015, Adam Fleming <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/214#discussion_r40031057>
> :
>
> > @@ -1,3 +1,4 @@
> > +[![Stories in Ready](https://badge.waffle.io/haiweiosu/cs5356.png?label=ready&title=Ready)](https://waffle.io/haiweiosu/cs5356)
>
> Surely you are not suggesting we put your badge on README.md for the class?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/214/files#r40031057>
> .
>


-- 
Sent from Gmail Mobile